build.gradle.kts

```diff
@@ -164,6 +164,8 @@ subprojects {
 
   java {
     toolchain {
+      // We set the vendor AMAZON OpenJDK
+      vendor.set(JvmVendorSpec.AMAZON)
```

Comment:
You should only set this in macOS and trino module. Also adding more comments and necessary documentations about this change.

Besides, you need to test when correto JDK is deleted in your environment, to see if it can be downloaded automatically.

Reply:
<img width="1624" alt="image" src="https://github.com/datastrato/gravitino/assets/15794564/a455d645-6be8-491e-bd51-aee4d66d4592">

Everything is Okay.